src/main/java/assets/game.js

```diff
@@ -1,5 +1,7 @@
 var isSetup = true;
 var isSonarPulse = false;
+var xMove = -1;

```

Comment:
```suggestion
var xMove = 0;
```
Initializing to 0 will make it clearer. Possible misinterpretation assumed with -1